bc/subscription/tasks.py

```diff
@@ -0,0 +1,22 @@
+from .models import FilingUpdate, Subscription
+
+
+def process_filing_update(pk):
```

Comment:
I have a convention of never (or almost never) calling parameters pk. Maybe:

```suggestion
def process_filing_update(fu_pk):
```

But FilingUpdate may get renamed `FilingWebhookEvent`, so, perhaps:

```suggestion
def process_filing_update(fwe_pk):
```

(Also, probably best not to call things F-U-underscore-anything.)